src/store/modules/governance/ethGovernance.ts

```diff
@@ -571,15 +571,22 @@ export class EthereumGovernance extends VuexModule.With({
 
     let metadata;
 
-    for await (const file of ipfs.get(hash, {
+    for await (const f of ipfs.get(hash, {
```

Comment:
This is done because TypeScript doesn't support Explicit type in a "for..of loop"
https://github.com/Microsoft/TypeScript/issues/3500